docs/reference/security.rst

```diff
@@ -115,26 +115,36 @@ either a super admin (``ROLE_SUPER_ADMIN`` or the role specified in the configur
 
 The permissions are:
 
-==========   ==================================================
+==========   ======================================================
 Permission   Description
-==========   ==================================================
+==========   ======================================================
 LIST         view the list of objects
 VIEW         view the detail of one object
 CREATE       create a new object
 EDIT         update an existing object
 DELETE       delete an existing object
 EXPORT       (for the native Sonata export links)
-ALL          grants LIST, VIEW, CREATE, EDIT, DELETE and EXPORT
-==========   ==================================================
+**ALL**      **grants LIST, VIEW, CREATE, EDIT, DELETE and EXPORT**
+==========   ======================================================
 
-Each permission is relative to an admin: if you try to get a list in FooAdmin (declared as ``app.admin.foo``
+Each permission is relative to an admin: if you try to get a list in ``FooAdmin`` (declared as ``app.admin.foo``
 service), Sonata will check if the user has the ``ROLE_APP_ADMIN_FOO_EDIT`` or ``ROLE_APP_ADMIN_FOO_ALL`` roles.
 
-The role name will be based on the name of your admin service. For instance, ``acme.blog.post.admin`` will become ``ROLE_ACME_BLOG_POST_ADMIN_{ACTION}``.
+.. note::
+
+    Declaring the same admin as `AppBundle\Admin\FooAdmin` results in
+    ``ROLE_APPBUNDLE\ADMIN\FOOADMIN_EDIT`` and ``ROLE_APPBUNDLE\ADMIN\FOOADMIN_ALL``!
```

Comment:
Uuuuh that's right. And it's ugly. Maybe we should do something about that.

Reply:
but how?

Reply:
its ugly, yes, but it is straight forward, regarding the admin service id

Reply:
In the meantime, let's document it, but something that allows to still see words might be better: `ROLE_AppBundle\Admin\FooAdmin_EDIT` will be simple, `ROLE_APP_BUNDLE_ADMIN_FOO_ADMIN` could be great. Being able to specify a prefix to remove like `AppBundle\Admin` and a suffix to remove like `Admin` could be a good idea too, leading to things like `ROLE_FOO_EDIT`

Reply:
sounds like a solution

Reply:
or several :P